### PR TITLE
Add optional Content Assignments param

### PR DESCRIPTION
**Pull request by e7mac**

### CODE REVIEW

**Comment by alexisoyama** [comment]
I'm confused. What's the purpose of this? Right now the flow is
1) Enable it.
2) Get data from Start / FCU and saves to VarCache..
and then, for what? 3) is missing for me.

**Comment by e7mac** [comment]
whoops.. sorry I missed your comment! This is a feature request we have from one of our clients. They want this to get insight into and to verify the content assignments that happen on the server. @alexisoyama

**Comment by e7mac** [comment]
Yup, through VarCache

**Comment by e7mac** [comment]
Good call! I'll add the public method in. Do you want me to do that for this PR or a new one?

We don't have to cache it, but I'll check with Product

**Comment by alexisoyama** [comment]
up to you. whichever that is easier for you

**Comment by e7mac** [comment]
We do wanna cache it. I'll add some of that in this PR

File: Leanplum-SDK/Classes/Leanplum.h [hunk 1]
```diff
@@ -608,6 +608,12 @@ typedef NS_ENUM(NSUInteger, LPTrackScreenMode) {
  */
 + (NSArray *)variants;
 
+/**
+ * Gets a list of A/B tests and the content assignments currently active for this user.
+ * Each A/B tests consists of and id, variant id and the variables assigned.
```
Comment (alexisoyama) [inline Leanplum-SDK/Classes/Leanplum.h]: typo. `of and`

File: Leanplum-SDK/Classes/Leanplum.m [hunk 2]
```diff
@@ -667,6 +667,16 @@ + (NSString *)platform
...
+{   [self startWithUserId:userId
+           userAttributes:attributes
+          responseHandler:startResponse
+includeContentAssignments:NO];
+}
+
++ (void)startWithUserId:(NSString *)userId
+         userAttributes:(NSDictionary *)attributes
+        responseHandler:(LeanplumStartBlock)startResponse
+includeContentAssignments:(BOOL)includeContentAssignments
```
Comment (alexisoyama) [inline Leanplum-SDK/Classes/Leanplum.m]: Is this feature for one customer or for everyone? 
I don't feel comfortable making a new Start just for enabling this flag.

File: Leanplum-SDK/Classes/Leanplum.m [hunk 3]
```diff
@@ -2174,6 +2191,11 @@ + (void)forceContentUpdate
 }
 
 + (void)forceContentUpdate:(LeanplumVariablesChangedBlock)block
+{
+    [self forceContentUpdate:block includeContentAssignments:NO];
+}
+
++ (void)forceContentUpdate:(LeanplumVariablesChangedBlock)block includeContentAssignments:(BOOL)includeContentAssignments
```
Comment (alexisoyama) [inline Leanplum-SDK/Classes/Leanplum.m]: Same here. FCU with just another flag doesn't seem to be a good approach. We have a lot of flags that customer can enable.

**Comment by e7mac** [comment]
Closing this in favor of #158

**Pull request closed without merging**